allenact/algorithms/onpolicy_sync/runner.py

```diff
@@ -810,30 +853,42 @@ def process_eval_package(
 
         num_tasks = pkg.num_non_empty_metrics_dicts_added
         metric_means = pkg.metrics_tracker.means()
+        callback_metric_means = dict()
+        tasks_callback_data = pkg.task_callback_data
 
         mode = pkg.mode
 
         if log_writer is not None:
             log_writer.add_scalar(
                 f"{mode}-misc/num_tasks_evaled", num_tasks, training_steps
             )
+            callback_metric_means[f"{mode}-misc/num_tasks_evaled"] = num_tasks
```

Comment:
Rather than repeating this string interpolation twice, can we assign it to a variable?
```python
            num_tasks_key = f"{mode}-misc/num_tasks_evaled"
            log_writer.add_scalar(num_tasks_key, num_tasks, training_steps)
            callback_metric_means[num_tasks_key] = num_tasks
```